java/integTest/src/test/java/glide/standalone/TransactionTests.java

```diff
@@ -92,6 +94,9 @@ public void test_standalone_transactions() {
         Transaction transaction = (Transaction) transactionTest(new Transaction());
         Object[] expectedResult = transactionTestResult();
 
+        transaction.select(0);
```

Comment:
lets make this test similar to the standalone test in python.

I will change the node test later to be similar

https://github.com/aws/glide-for-redis/blob/fa625a97417ac6bee00512f7fa93cf99dbdda907/python/python/tests/test_transaction.py#L275

Reply:
Try error case too, like
```
127.0.0.1:6379> SELECT -1
(error) ERR DB index is out of range
```